<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterSelectorClass</class>
 <widget class="QWidget" name="FilterSelectorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FilterSelector</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="5">
    <widget class="QListWidget" name="allFiltersWidget">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="useButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Use</string>
     </property>
     <property name="icon">
      <iconset resource="../../resources/main.qrc">
       <normaloff>:/new/prefix1/plus.png</normaloff>:/new/prefix1/plus.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="5">
    <widget class="QListWidget" name="filtersWidget"/>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="deleteButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Del</string>
     </property>
     <property name="icon">
      <iconset resource="../../resources/main.qrc">
       <normaloff>:/new/prefix1/minus.png</normaloff>:/new/prefix1/minus.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="upButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../resources/main.qrc">
       <normaloff>:/new/prefix1/up_alt.png</normaloff>:/new/prefix1/up_alt.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="downButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../resources/main.qrc">
       <normaloff>:/new/prefix1/down_alt.png</normaloff>:/new/prefix1/down_alt.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../resources/main.qrc"/>
 </resources>
 <connections/>
</ui>
